src/amo/actions/reviews.js

```diff
@@ -1,15 +1,18 @@
 /* @flow */
 import {
-  SHOW_EDIT_REVIEW_FORM,
-  SHOW_REPLY_TO_REVIEW_FORM,
   FETCH_REVIEWS,
   HIDE_EDIT_REVIEW_FORM,
   HIDE_REPLY_TO_REVIEW_FORM,
   SEND_REPLY_TO_REVIEW,
+  SEND_REVIEW_FLAG,
   SET_ADDON_REVIEWS,
   SET_REVIEW,
   SET_REVIEW_REPLY,
+  SET_REVIEW_WAS_FLAGGED,
```

Comment:
Nitpick but if this were clearer like `SET_REVIEW_MARKED_ABUSIVE` that might be nice.